.gitignore

```diff
@@ -114,6 +114,7 @@ node_modules/
 /ajax/libs/strophe.js/release-1.2.0
 /ajax/libs/raphael/semver/
 /ajax/libs/clappr/media_control_youtube/
+/ajax/libs/clarity-icons/0.7.5-dev
```

Comment:
Just follow https://github.com/cdnjs/cdnjs/commit/94ea88b85054a45ca6d5b1f6ffc190eea2cedc78
I'll update this like https://github.com/cdnjs/cdnjs/commit/e61dc9c9271fc82e5d7e5260cedbaaf4efbb01f0